build/nuget-release.yml

```diff
@@ -1,5 +1,13 @@
 name: $(Package.Version)
 
+parameters:
+  - name: 'Package.Version'
+    type: 'string'
+  - name: 'Prerelease'
+    displayName: 'Prerelease string (ex. -alpha, -alpha1, -BETA, -update20171020) or none'
+    type: 'string'
+    default: 'none'
```

Comment:
Do we have some logic somewhere that tests on 'none' ?  To make sure that nothing is added when the value is 'none' ?

Reply:
Oh, this is actually not needed; will remove it. 👍